pytorch_lightning/plugins/training_type/tpu_spawn.py

```diff
@@ -179,6 +175,14 @@ def reduce_early_stopping_decision(self, should_stop: bool) -> bool:
         should_stop = int(stop.item()) == self.world_size
         return should_stop
 
+    def reduce(self, output, group: Optional[Any] = None, reduce_op: str = None):
```

Comment:
this effectively means that only sum or mean is valid `reduce_op` right?

Reply:
For now, yes.

Reply:
so let's add there assert 
```
assert reduce_op in (None, 'sum', 'mean')
```